kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/HelmClient.java

```diff
@@ -152,4 +162,38 @@ private String mapToValuesParameterArgument(Map<String, String> m) {
     private static String concatEntry(Map.Entry<String, String> entry) {
         return String.format("%s=%s", entry.getKey(), entry.getValue());
     }
+
+    /**
+     * Gets chart version.
+     *
+     * @param chartName the chart name
+     * @param releaseVersion the release version
+     * @return the chart versions
+     */
+    public String getChartVersion(String chartName, String releaseVersion) {
+        String chartVersion = "";
+        if (releaseVersion != null && !releaseVersion.isEmpty()) {
+            LOGGER.info("Getting chart version for {}", chartName);
+            ExecResult result = Exec.exec(null, command("search", "repo",
```

Comment:
`helm search repo <name> --versions` is an inexact match:

```
hashicorp/vault                 	0.6.0        	1.4.2      	Official HashiCorp Vault Chart
hashicorp/vault                 	0.5.0        	           	Install and configure Vault on Kubernetes.
hashicorp/vault                 	0.4.0        	           	Install and configure Vault on Kubernetes.
hashicorp/vault-secrets-operator	0.4.0        	0.4.0      	Official Vault Secrets Operator Chart
hashicorp/vault-secrets-operator	0.3.4        	0.3.4      	Official Vault Secrets Operator Chart
```

so you need to make sure that you process only the charts you are interested in.